frontend/sass/laletterbuilder/variables.scss

```diff
@@ -0,0 +1,14 @@
+// SPACING
```

Comment:
This is great! I wonder if it's worth calling this file `spacing.scss`, and keeping it to just these variables. I can imagine that making it easier to find/change spacing if all the info is in its own file.